### PR TITLE
V8 package update

### PR DESCRIPTION
**Pull request by tudorms**
### Type of Change
- Bug fix (non-breaking change which fixes an issue)

### Why
Picking up an updated V8 package version and fix the projects that broke because we're not actively testing this configuration.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/10766)

### CODE REVIEW

File: vnext/Microsoft.ReactNative.Cxx/JSI/NodeApiJsiRuntime.h [hunk 1]
```diff
@@ -5,13 +5,14 @@
 #ifndef MICROSOFT_REACTNATIVE_JSI_NODEAPIJSIRUNTIME
 #define MICROSOFT_REACTNATIVE_JSI_NODEAPIJSIRUNTIME
 
+// Standard Library
+#include <cstdint>
```
Comment (vmoroz) [inline vnext/Microsoft.ReactNative.Cxx/JSI/NodeApiJsiRuntime.h:9]: I wonder why this change is required. The <cstdint> must be included through the js_native_api.h included by the js_native_ext_api.h

Comment (tudorms) [reply vnext/Microsoft.ReactNative.Cxx/JSI/NodeApiJsiRuntime.h:9]: This was needed because the ReactNativePicker project (an external npm package thing used in Playground) was building NodeApiJsiRuntime.cpp through a vcxitems import. I'll try to find a cleaner alternative there.
